### DISCUSSION TITLE
No module named 'tkinter'

### DISCUSSION THREAD

**Discussion by tqye2000** [beeware/briefcase#2078] [Q&A]
Has anyone tried to build and run an application involving tkinter on Windows?
I have an application which runs fine with `briefcase dev` . However when I built it as a Windows executable, and ran it, it throwed _ModuleNotFoundError: No module named 'tkinter'_

**Comment by tqye2000** [comment]
I found a solution to this issue.
https://stackoverflow.com/questions/37710205/python-embeddable-zip-install-tkinter/73761645#73761645

Basically you could manually copy the following:
- **tcl** folder to **\your_app\build\your_app\windows\app\sr**c

- **tkinter** folder (which is under Lib) to **\your_app\build\your_app\windows\app\src\app_packages** folder

- file **_tkinter.pyd** to the **\your_app\build\your_app\windows\app\src\app_packages** folder

- files **tcl86t.dll tk86t.dll zlib1.dll** (which are under DLLs, zlib1.dll) to **\your_app\build\your_app\windows\app\src** folder

**Reply by rmartin16** [reply]
- linking: https://github.com/beeware/briefcase/issues/383
